src/controller/CHIPClusters.h

```diff
@@ -72,23 +71,6 @@ class DLL_EXPORT BasicCluster : public ClusterBase
     CHIP_ERROR ReadAttributeClusterRevision(Callback::Callback<> * onCompletion);
 };
 
-class DLL_EXPORT BindingCluster : public ClusterBase
-{
-public:
-    BindingCluster() : ClusterBase(kBindingClusterId) {}
-    ~BindingCluster() {}
-
-    // Cluster Commands
-    CHIP_ERROR Bind(Callback::Callback<> * onCompletion, chip::NodeId nodeId, chip::GroupId groupId, chip::EndpointId endpointId,
-                    chip::ClusterId clusterId);
-    CHIP_ERROR Unbind(Callback::Callback<> * onCompletion, chip::NodeId nodeId, chip::GroupId groupId, chip::EndpointId endpointId,
-                      chip::ClusterId clusterId);
-
-    // Cluster Attributes
-    CHIP_ERROR DiscoverAttributes(Callback::Callback<> * onCompletion);
-    CHIP_ERROR ReadAttributeClusterRevision(Callback::Callback<> * onCompletion);
-};
-
```

Comment:
I"m pretty sure the `Binding Cluster` should stay here.

I do wonder if there is something wrong with your `all-clusters-app.zap` file since `MfgSpecificPing` has disappeared too.

Reply:
I'm courious about this, however, after I checked the zap repo, this cluster is removed, please update related file or document so others who touches these file can generate the same files as you.

Reply:
@erjiaqing you should use the script `/scripts/tools/zap_generate.sh` or `/scripts/tools/zap_configure.sh`. If not you won't have any of the CHIP specific cluster as ZAP is a ZCL configurator that works for both Zigbee and CHIP stack depending of the configuration files that you input at start. Using the scripts saves you the trouble of manually linking the templates and the zcl.json files needed for it to works with CHIP. Also the zap repo doesn't contain anything that is CHIP specific.